phpstan-baseline.neon

```diff
@@ -0,0 +1,6 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Parameter \\#3 \\$toxicity of method Ihsw\\\\Toxiproxy\\\\Proxy\\:\\:create\\(\\) expects string, float given\\.$#"
```

Comment:
Nebylo lepší to patchnout, když už to někde děláme? https://github.com/keboola/db-extractor-common/blob/fb5d6cec9d762085e7db62817b3fca6b16e3c32f/patches/ihsw-toxiproxy-php-client-Proxy.php.patch#L4

Reply:
Nebo ještě líp forknout a fixnout tam. To repo je mrtvý od 2015, takže nevidím důvod nemít fork.

Reply:
jasně, patchnul jsem to